docs/extensions-plugins/sublime.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Pieces for Developers Sublime Plugin
+description: Learn about the Pieces Sublime Extension, how it functions, and all you can do with the Sublime Plugin.
+---
+import { pieces_app } from "/src/lib/const";
+import CTAButton from "/src/components/CTAButton";
+import GridFlexRow from "/src/components/GridFlexRow";
+
+# [Pieces for Developers](https://pieces.app) | Sublime Plugin
+
+![Pieces Sublime Plugin](https://github.com/Bishoy-at-pieces/plugin_sublime/assets/167128903/272cf4cc-60ff-4bd4-8701-3e3feb38a8b1)
+
+
+##### <p align="center"> [Website](https://pieces.app/) • [Pieces OS Documentation](https://docs.pieces.app/) • [Pieces Sublime github repository](https://github.com/pieces-app/plugin_sublime)
+</p>
+
+# Important
```

Comment:
Can we keep the intro section similar to the rest of the extensions: 

For reference: https://github.com/pieces-app/documentation/blob/sublime-package/docs/extensions-plugins/jetbrains.mdx : 
In order to use the Pieces JetBrains Plugin, you must have [Pieces OS.](https://github.com/pieces-app/documentation/blob/sublime-package/installation-getting-started/what-am-i-installing) We recommend also using the Pieces for Developers desktop app.

:::

You can install the JetBrains plugin through the [JetBrains Marketplace](https://github.com/pieces-app/documentation/blob/sublime-package/docs/extensions-plugins/%7Bpieces_app.links.extensions.jetbrains%7D) or by searching for Pieces in the plugins search of your IDE.

Once you've installed the plugin, you can begin using Pieces without restarting your IDE.

Reply:
+ The GIFs are not rendering in the README

Reply:
In general can you try to make the structure and heading similar to the rest of the extensions to maintain similarity wherever possible @Bishoy-at-pieces

Reply:
Yea same for the README on the sublime repo I think I might record another videos